backend/Functions/Edna.AssignmentLinks/Edna.AssignmentLinks/AssignmentLinksApi.cs

```diff
@@ -62,6 +62,7 @@ public AssignmentLinksApi(ILogger<AssignmentLinksApi> logger, IMapper mapper)
 
             IEnumerable<AssignmentLinkDto> assignmentLinkDtos = sameAssignmentLinks
                 .OrderBy(linkEntity => linkEntity.Timestamp.Ticks)
+                .Where(linkEntity => Guid.TryParse(linkEntity.RowKey, out Guid result))
```

Comment:
Does it make sense to move this into `_mapper`? That way it should help in case newer entry points for LinkEntity -> LinkDto conversion popup in future.

Reply:
I have made some changes in this regard: essentially, the mapper while converting `LinkEntity` to `LinkDto` will map the Id only if it can be parsed to a `Guid`, else it will keep it as the default `Guid.Empty` value. 

On the client-side, I have added a check for filtering such links with `Guid.Empty` Ids, indicating that these are not "well-formed". Although the same holds here also: the check might have to be propagated in the client-side if a similar condition arises somewhere else. But given the store can be re-used where ever we want, in future, it is less likely we will have to rewrite such conditions. Any thoughts?